homeassistant/components/bluetooth/active_update_coordinator.py

```diff
@@ -0,0 +1,164 @@
+"""A Bluetooth passive coordinator that receives data from advertisements but can also poll."""
+from __future__ import annotations
+
+from collections.abc import Callable, Coroutine
+import logging
+from typing import Any, Generic, TypeVar
+
+from bleak import BleakError
+
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.debounce import Debouncer
+from homeassistant.util.dt import monotonic_time_coarse
+
+from . import BluetoothChange, BluetoothScanningMode, BluetoothServiceInfoBleak
+from .passive_update_coordinator import PassiveBluetoothDataUpdateCoordinator
+
+POLL_DEFAULT_COOLDOWN = 10
+POLL_DEFAULT_IMMEDIATE = True
```

Comment:
_(I don't know your coordinator well, so disregard if this isn't relevant.)_

These are the same values for the default coordinator debouncer; is this a redefinition or something else?

https://github.com/home-assistant/core/blob/ace20782f1f8a498e333c807135a8f782222c3b1/homeassistant/helpers/update_coordinator.py#L28-L29

Reply:
They happen to be the same right now but we might end up bumping them up because BLE connection costs are usually 100x more expensive than IP so I didn't want to tie it to the base coordinator